tests/transaction_test.move

```diff
@@ -75,14 +73,12 @@ fun op_return_script_tests() {
     ];
     let expected_result = vector[x"68656c6c6f20776f726c64", x"", x"01020304", x"010203", x"010203"];
 
-    let mut i = 0;
-    while (i < data.length()) {
+    data.length().do!(|i| {
         let o = &parse_output(0, data[i]);
         // this return error code at test index fails
-        assert!(o.is_op_return(), i);
-        assert!(o.op_return() == expected_result[i], i);
-        i = i + 1;
-    };
+        assert!(o.is_op_return());
```

Comment:
use `assert_eq`

Reply:
If the value we need to check has a boolean type. This makes more sense if we just use assert!

Reply:
No, i think it still better the `assert_eq!(o.op_return(), true)` . It immidietly tells the user we are expecting true, assert also checks for 1 and 0 etc. Lets update it to keep it consistent

Reply:
update